README.md

```diff
@@ -7,7 +7,7 @@
 ![screenshot](assets/desktopv.JPG)
 
 This project is to create a mobile version skeleton using Figma template
-It is a web page that contains three sections in the first section we have a header, the main page, the second section contain works page, about page, and the last section contain a contact form and CSS to style it. I added index.html file and style.css file, test.md file, and test.md file to .gitignore file.
+It is a web page that contains three sections in the first section we have a header, the main page, the second section contains works page, about page, and the last section contains a contact form and CSS to style it. I added index.html file and style.css file, test.md file, and test.md file to .gitignore file.
 
 ## Live Demo
```

Comment:
- [ ] Your hosted application on GitHub pages appears to overlap on mobile screen. From the screenshot below of your design we can see that some portions of the your design is not displayed on the screen viewport, and therefore does not fit the screen width as intended. We actually want our want our design to look closely as the chosen [Figma template](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&scaling=min-zoom&page-id=0%3A1). Please ensure to improve upon the mentioned aspects.

**Your mobile design:**
![image](https://user-images.githubusercontent.com/12745474/127041556-342c0e7f-ae72-4b46-b169-d651cbfacda7.png)

**The original mobile design:**
![image](https://user-images.githubusercontent.com/12745474/127042125-0f98871f-3892-45e8-975b-ce3ac091b02f.png)